docs/components/motor/roboclaw.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Configure a roboclaw motor"
+linkTitle: "roboclaw"
+weight: 22
+type: "docs"
+description: "Configure a brushed DC motor driven by a roboclaw motor controller."
+images: ["/components/img/components/motor.svg"]
+# SMEs: Olivia, Rand
+---
+
+The `roboclaw` model of the motor component supports [standard brushed DC motors](https://en.wikipedia.org/wiki/DC_motor) driven by [Basicmicro's](https://www.basicmicro.com/) [RoboClaw](https://www.basicmicro.com/RoboClaw-2x30A-Motor-Controller_p_9.html) motor controller.
+
+{{< alert title="Note" color="note" >}}
+
+You must set up your RoboClaw before configuring it.
+Follow [this guide](https://resources.basicmicro.com/roboclaw-motor-controllers-getting-started-guide/) to do so.
+
+{{< /alert >}}
+
+To configure a `roboclaw` motor as a component of your robot:
+
+{{< tabs name="Configure your roboclaw motor">}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your motor, select the type `motor`, and select the `roboclaw` model.
+
+Click **Create component**.
+
+![A roboclaw motor config.](../../img/motor/roboclaw-ui-config.png)
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json
+{
+  "components": [
+    {
+      "name": <"your-roboclaw-motor-name">,
+      "type": "motor",
+      "model": "roboclaw",
+      "attributes": {
+        "serial_path": <"your-serial-path">,
+        "motor_channel": <#>,
+        "serial_baud_rate": <#>
```

Comment:
```suggestion
        "motor_channel": <int>,
        "serial_baud_rate": <int>
```
[opt nit] It seems that this is quite a bit more common in the docs right now

Reply:
Ah yes, good catch!